### ISSUE TITLE
Column default value with multiple functions

### ISSUE THREAD

**Issue by JeongJuhyeon** [sequelize/sequelize-auto#493]
We would like to use a column with a default value such as `year(current_timestamp())`. What we are seeing right now is that the parentheses are getting removed, result in "yearcurrent_timestamp".

**Comment by steveschmitt** [comment]
Yes, we need to fix the current way of converting from the server-side function to the default value.

If you hand-code the model definition, what is the correct way?  Should it be like:
```js
    testyear: {
      type: DataTypes.INTEGER,
      defaultValue: Sequelize.fn('year(current_timestamp())'),
    },
```
Or something else?  Can you try it and tell me what works at run-time?

**Comment by steveschmitt** [comment]
@JeongJuhyeon Can you tell me what is the correct output?
